src/adapter/basic.ts

```diff
@@ -215,6 +221,10 @@ export class Adapter<CustomSettingsDefinition extends SettingsDefinitionMap = Se
   initializeDependencies(inputDependencies?: Partial<AdapterDependencies>): AdapterDependencies {
     const dependencies = inputDependencies || {}
 
+    if (this.config.settings.LOG_CENSOR === false) {
+      process.env['LOG_CENSOR'] = 'false'
+    }
```

Comment:
I don't get this part, the settings will be sourced from the env vars, why are we setting them back?